src/scripts/templates/validate.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+
+validate_numeric = '''
+function validate_numeric(test_value, expected_result, false_results) {
+    var radix = 2 + Math.abs(expected_result % 35);
```

Comment:
Try to avoid using magic numbers! (Create or use a predefined variable, and a description instead)

Reply:
I would not call them magic numbers since the standard uses these numbers as well. See the corresponding part of the standard [here](https://www.ecma-international.org/ecma-262/5.1/#sec-15.7.4.2).

Moreover, wrapping these numbers to variables/constants does not reduce the real count of "magic numbers".